packages/cozy-mespapiers-lib/src/components/Papers/Empty/Empty.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import UiEmpty from 'cozy-ui/transpiled/react/Empty'
+
+import HomeCloud from '../../../assets/icons/HomeCloud.svg'
+import EmptyWithConnector from './EmptyWithConnector'
+
+const Empty = ({ connector, accounts }) => {
```

Comment:
`cozy-ui` has a component with the exact same name, could it be an issue?

Reply:
non on le fait déjà, on regarde les sources d'import pour vérifier, et je n'ai pas d'idée de meilleur nom